portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
add a function doc to describe this function.

Reply:
Done